lib/ObjWriter/objwriter.cpp

```diff
@@ -249,6 +252,58 @@ extern "C" void FinishObjWriter(ObjectWriter *OW) {
   delete OW;
 }
 
+extern "C" void CreateDataSection(ObjectWriter *OW, const char *SectionName,
+                                  bool ReadOnly)
+{
```

Comment:
{ starts the end of function declaration per LLILC coding convention (which aligns with LLVM).
I think the following code does not follow this either.
Please run codeformatter tool to fix this -- https://github.com/dotnet/llilc/blob/master/Documentation/Code-Formatting.md
Or PR run will fail. cc, for details --  @swaroop-sridhar